### DISCUSSION TITLE
Unable to translate the audio file

### DISCUSSION THREAD

**Discussion by jay-pixster** [deepgram/community#153] [General help]
I have tried uploading a 3min audio file to deepgram. The audio is in English. When I am trying to get the transcribed file in Spanish, I am not getting any response. Or I am getting a response of only 2-3 words. Is there something that I am doing wrong? What can I do to resolve this?

**Comment by nikolawhallon** [comment]
Can you give more details how you are making the request? What parameters/query string/full url.

If the file contains English audio and you are requesting transcription in Spanish, Deepgram will try to listen for Spanish language being spoken, and probably not understand much since the audio was in English. If you are looking to have the file translated into Spanish, we do have a beta translation feature, you could do something like `https://api.beta.deepgram.com/v1/listen?translate=es`
